net-libs/mbedtls/mbedtls-3.6.3.1.ebuild

```diff
@@ -48,13 +51,9 @@ src_prepare() {
 	use threads && enable_mbedtls_option MBEDTLS_THREADING_C
 	use threads && enable_mbedtls_option MBEDTLS_THREADING_PTHREAD
 
-	cmake_src_prepare
-}
+	sed -i -e "s:VERSION 3.5.1:VERSION 3.10:g" CMakeLists.txt || die
 
-src_configure() {
-	# Workaround for https://github.com/Mbed-TLS/mbedtls/issues/9814 (bug #946544)
```

Comment:
Are you sure this is fixed in 3.6? https://github.com/Mbed-TLS/mbedtls/pull/10179 is open still.